cypress/integration/seededFlows/articleFlows/postModerationTools.spec.js

```diff
@@ -151,22 +151,109 @@ describe('Moderation Tools for Posts', () => {
         });
       });
     });
+
+    it('should show Suspend User button for an unsuspended user', () => {
+      cy.get('@moderatorUser').then((user) => {
+        cy.loginAndVisit(user, '/series_user/series-test-article-slug').then(
+          () => {
+            cy.findByRole('button', { name: 'Moderation' }).click();
+
+            cy.getIframeBody('[title="Moderation panel actions"]').within(
+              () => {
+                cy.findByRole('button', { name: 'Open admin actions' })
+                  .as('moderatingActionsButton')
+                  .pipe(click)
+                  .should('have.attr', 'aria-expanded', 'true');
+
+                cy.findByRole('button', {
+                  name: 'Suspend series_user',
+                }).should('exist');
+              },
+            );
+          },
+        );
+      });
+    });
+
+    it('should not suspend the user when no suspension reason given', () => {
```

Comment:
@aitchiss In this and the nest tests, I repeat the opening of the Moderating actions panel; is there a way to avoid the repetition? Maybe create a sub-context block?